packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -222,6 +249,16 @@ export function ArticleMetadataPanel({
                     setAuthorsFilter('')
                   }}
                   block
+                  renderExtraFooter={() =>
+                    authorsFilter &&
+                    data?.authors.nodes.length === 0 && (
+                      <div style={{float: 'right', margin: '20px'}}>
+                        <Button onClick={() => handleCreateAuthor()} appearance={'primary'}>
+                          {t('authors.panels.createAuthor')}
```

Comment:
Please make the label of the button more descriptive. "Create Auhtor Profile for %Name%"